src/cesar/simplify.rs

```diff
@@ -10,16 +10,17 @@ use crate::cesar::pass6::Pass6;
 use crate::cesar::pass7::Pass7;
 use crate::cesar::pass8::Pass8;
 use crate::cesar::pass9::Pass9;
+use crate::cesar::base_pass::BasePass;
```

Comment:
Doublecheck if this is required. I don't see why it should be but have not tried removing it and checking it the code still compiles. Ignore this comment if you have already checked that this is required.

Reply:
Already checked. It is required.